packages/contracts/src/dollar/facets/CollectableDustFacet.sol

```diff
@@ -3,9 +3,8 @@ pragma solidity ^0.8.19;
 
 import "../libraries/LibCollectableDust.sol";
 import {Modifiers} from "../libraries/LibAppStorage.sol";
-import {ICollectableDust} from "../../dollar/interfaces/utils/ICollectableDust.sol";
 
-contract CollectableDustFacet is ICollectableDust, Modifiers {
+contract CollectableDustFacet is Modifiers {
```

Comment:
Why was the `ICollectableDust` interface removed? I think it is crucial that `CollectableDustFacet` should implement `ICollectableDust`.

Reply:
> Why was the `ICollectableDust` interface removed? I think it is crucial that `CollectableDustFacet` should implement `ICollectableDust`.

ohh imma add it then